components/widgets/modals/full_screen_modal.tsx

```diff
@@ -12,7 +12,14 @@ import './full_screen_modal.scss';
 // This must be on sync with the animation time in ./full_screen_modal.scss
 const ANIMATION_DURATION = 100;
 
-export default class FullScreenModal extends React.Component {
+type Props = {
+    show: boolean;
+    onClose: () => void;
+};
+
+type State = {};
+
+export default class FullScreenModal extends React.Component<Props, State> {
```

Comment:
no need for empty state. Just don't pass the second arg to React.Component

Reply:
Right